1.0/gradle/vividus-test.gradle

```diff
@@ -134,6 +141,14 @@ tasks.withType(StoriesExec).configureEach {
     }
     ignoreExitValue true
     doLast {
+        def pathToExitCodeFile = project.hasProperty('fileToSaveExitCode') ? new groovy.text.SimpleTemplateEngine().createTemplate(project.fileToSaveExitCode)
+                                                                                                                   .make(project.getProperties())
+                                                                                                                   .toString()
+                                                                           : fileToSaveExitCode
+        if (pathToExitCodeFile) {
+            logger.warn("Saving exit code to a file: ${pathToExitCodeFile}")
```

Comment:
isn't `info` enough?

Reply:
<img width="1049" alt="image" src="https://user-images.githubusercontent.com/17816729/198629385-ed4fbd8a-fe6d-48f1-b6cd-93d7900e0f2c.png">
 info is not printed by default, still changed to lifecycle

Reply:
>  still changed to lifecycle

are changes lost?